src/translate-virtual-address.h

```diff
@@ -187,39 +211,39 @@ static NO_INLINE bool translate_virtual_address(STATE_ACCESS &a, uint64_t *ppadd
         if (xwr != 0) {
             // These protection bit combinations are reserved for future use
             if (unlikely(xwr == PTE_XWR_W_MASK || xwr == (PTE_XWR_W_MASK | PTE_XWR_X_MASK))) {
-                return false;
+                break;
             }
-            // (We know we are not PRV_M if we reached here)
-            if (priv == PRV_S) {
-                if ((pte & PTE_U_MASK)) {
-                    // S-mode can never execute instructions from user pages, regardless of the state of SUM
```

Comment:
This whole code section is just gone, I se no if for `TRANSLATION_HS`, I think there were lost while you rebased. To confirm this, I noticed the new `translate_vaddr.S` tests that I created to make this file 100% covered before hypervisor are failing:

```
translate_vaddr.bin: failed
translate_vaddr.bin: returned halt payload 1, expected 0
translate_vaddr.bin: terminated with mcycle = 159, expected 347
```